ui/controls.js

```diff
@@ -892,6 +892,36 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     this.controlsContainer_.appendChild(this.daiAdContainer_);
   }
 
+
+  /**
+   * Adds a seekbar depending on the configuration.
+   * By default an instance of shaka.ui.SeekBar is created
+   * This behaviour can be overriden by providing a SeekBar factory using the 
+   * registerSeekBarFactory function.
+   * 
+   * @private
+   */
+  addSeekBar_(){
+
+    if (this.config_.addSeekBar) {
+      
+      if( shaka.ui.ControlsPanel.seekBarFactory_ == null )
+      {
+        this.seekBar_ = new shaka.ui.SeekBar(this.bottomControls_, this);
+      }
+      else{
```

Comment:
There are many style violations the linter will complain about in this PR.  Please run `python build/check.py` to run the smallest set of linter and compiler checks, or `python build/all.py` to run all lint & build steps.

Reply:
My bad, it should be easy to fix !